x/wasm/client/cli/tx.go

```diff
@@ -57,6 +57,9 @@ func StoreCodeCmd() *cobra.Command {
 		Args:    cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
```

Comment:
Looks like something a linter should have caught.

Reply:
yes, this was caught by the linter. I marked this and other commands in #645.